polybar-scripts/player-mpv/mpv-control.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
```

Comment:
Can we include the function of this script into the python script as command line parameters? Then we only have one script file.

Reply:
Technically we can, but that leads to code complexity (btw, player-mpv.py code already too branchy). Also, the reason mpv-control.sh is a separate bash script is to load it as quick as posible. For operations like time-pos (seek forward/backword), which by default happends on mouse scroll, the speed is important.

Reply:
`player-mpris-tail` is 4 times longer, I don't think it's too complex.

Is the speed really noticable?

Reply:
Will check soon.

Reply:
Would be perfect! :+1:

Reply:
The speed isn't noticable. I didn't know python is fast😃

Reply:
Speed is relative. We work with scripts here. They are fast and easy to customize. If we would need execution speed, we would do things in C.  :+1: